frontend/ios/DroidKaigi 2019/Assets.xcassets/alert.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "exclamation-mark.png",
```

Comment:
I added temporary image resources. Is it possible to get resources same as Android as png images...?

Reply:
@MihoSasaki 
Could you please use icons in https://drive.google.com/open?id=1R6lEMomFePkfWtmoCwfIfR2c7N3qc9h7 ? Also, it's okay to do it on another PR.

Reply:
I applied them to both prs🐱